ui/pages/v2/GitRepositoryDetail.tsx

```diff
@@ -0,0 +1,47 @@
+import * as React from "react";
+import styled from "styled-components";
+import Interval from "../../components/Interval";
+import Page, { Content, TitleBar } from "../../components/Page";
+import SourceDetail from "../../components/SourceDetail";
+import {
+  GitRepository,
+  SourceRefSourceKind,
+} from "../../lib/api/core/types.pb";
+
+type Props = {
+  className?: string;
+  name: string;
+  namespace: string;
+};
+
+function GitRepositoryDetail({ className, name, namespace }: Props) {
+  return (
+    <Page error={null} className={className}>
+      <SourceDetail
+        name={name}
+        namespace={namespace}
+        type={SourceRefSourceKind.GitRepository}
+        info={(s: GitRepository) => ({
+          URL: s.url,
+          Ref: s.reference.branch,
+          "Last Updated": "",
+          Interval: <Interval interval={s.interval} />,
+          Cluster: "",
+          Namespace: s.namespace,
+        })}
+      ></SourceDetail>
+    </Page>
+  );
+}
+
+export default styled(GitRepositoryDetail).attrs({
+  className: GitRepositoryDetail.name,
+})`
+  ${TitleBar} {
+    margin-bottom: 0;
+  }
```

Comment:
What was happening without this override?

Reply:
Extra margin on `TitleBar`. It assumes that there is a title, and in this case I am keeping the title in the component instead of the page.

Reply:
Mmmmmm I see 🌞